### ISSUE TITLE
Contributor License Agreement signing broken

### ISSUE THREAD

**Issue by MichaelSteinecke** [dotnet-foundation/website#731]
Hello,
I want to sign the Contributor License Agreement. However one site is not working, the other has a corrupt SSL cert. I'm not sure, where to report this, but this is the most appropriate place, I could find.

Not working (for me):
https://cla.dotnetfoundation.org/

Certificate error:
https://cla2.dotnetfoundation.org/

**Comment by clairernovotny** [comment]
Hi,

The certificate on https://cla.dotnetfoundation.org/ seems to be working for me; what is pointing you to cla2?

**Comment by MichaelSteinecke** [comment]
Yeah, on https://cla.dotnetfoundation.org/ the certificate is working. However. I'm not able to manage to sign something there. My only Option is to click on Administrator Sign In. Doing so, results in GitHub requesting me to allow my account usage. After confirmation I'm redirected back to https://cla.dotnetfoundation.org/. Thats all.

I found https://cla2.dotnetfoundation.org/ on some places mentioned therefore I tried that one as well. For example on:
https://github.com/dotnet-foundation/foundation/tree/master/governance

**Comment by MichaelSteinecke** [comment]
From this blog:
https://adrientorris.github.io/github/how-to-sign-microsoft-cla-on-github.html

**Comment by MichaelSteinecke** [comment]
The whole DocuSign part enpictured there is missing for me, on https://cla.dotnetfoundation.org/ (Tried Chrome, Edge, IE, Opera on two devices)

**Comment by clairernovotny** [comment]
There's no end-user access to the CLA. What are you trying to do?

**Comment by MichaelSteinecke** [comment]
I wanted to post an issue to AutoMapper
https://github.com/AutoMapper/AutoMapper/issues
It seems, it is required to sign the CLA beforehand, before posting there. At least the submit button is inactive to me. They claim it is required to fill the CLA before creation of an PR. My guess is, that is the reason preventing submitting the issue. https://github.com/AutoMapper/AutoMapper/blob/master/CONTRIBUTING.md

I could find no mention of an restriced target audiance of the CLA. And how do you define an end user?

**Comment by ViliusS** [comment]
Just hit the same issue myself. If cla2 host is not used anymore .NET foundation should make a redirect to cla or just make a static page on cla2 with explanation what's going on and how to sign CLA now. cla2 link is used in dozen of projects on GitHub so just dropping the host without explanation is not acceptable.
